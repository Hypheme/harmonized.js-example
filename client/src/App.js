import React, { Component } from 'react';
import './App.css';
import ToDos from './todos';
import Authors from './authors';

class App extends Component {
  componentDidMount() {}

  render() {
    return (
      <div className="App">
        <div>
          <Authors />

          <ToDos />
          
        </div>
      </div>
    );
  }
}

export default App;
