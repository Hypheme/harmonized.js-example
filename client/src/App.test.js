import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import App from './App';

test('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

test('displays loading message', () => {
  const app = renderer.create(<App/>);
  expect(app).toMatchSnapshot();
});
