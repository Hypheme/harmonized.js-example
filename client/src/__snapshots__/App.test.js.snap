// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays loading message 1`] = `
<div
  className="App"
>
  <div>
    <h1>
      Message
    </h1>
    <div>
      no message yet
    </div>
  </div>
</div>
`;
